nucleus-api/src/main/java/io/github/nucleuspowered/nucleus/api/service/NucleusKitService.java

```diff
@@ -21,6 +22,12 @@
     Set<String> getKitNames();
 
     /**
+     * Gets all the kits currently in NucleusPlugin.
+     * @return A {@link Map} of names to {@link Kit}s.
```

Comment:
The API is meant to be Map so the word will have to do. See below convo.
```
Mohron - Today at 6:16 PM
The only other question I'd have is why the ImmutableMap for getKits?
dualspiral - Today at 6:19 PM
To prevent plugins absent mindedly deleting kits by accident when doing filtering on the map.
Mostly a safety thing.
Mohron - Today at 6:20 PM
the API spec can just be map ya?
dualspiral - Today at 6:20 PM
Yep.
```